lib/rails/generators/task/templates/task.rb

```diff
@@ -2,18 +2,17 @@
 # 1. Re-runnable on production?
 # 2. Is there a chance emails will be sent?
 # 3. puts ids & logs (progress log)
-# 4. Can you update the records with an update all instead of instantizing?
+# 4. Should this be inside a Active Record Transaction block?
 # 5. Are there any callbacks?
 # 6. Performance issues?
-# 7. Scoping to account
```

Comment:
let's discuss these points once ?

Reply:
Cool. These were present in the existing gem.